packages/document/docs/zh/config/options/output.mdx

```diff
@@ -0,0 +1,149 @@
+
+- **normal:** 在构建产物目录中生成一个 `.rsdoctor` 文件夹，其中包含各种数据文件，并在报告页面中展示代码。输出目录可以通过 [reportDir](#reportdir) 进行配置。
+- **brief:** 在构建产物目录的 `.rsdoctor` 文件夹中生成一个 HTML 报告文件，所有构建分析数据会整合注入到这个 HTML 文件中，可以通过浏览器打开该 HTML 文件查看报告。brief 模式还有更多配置项，详细信息请参阅：[brief](#brief)。
+
+> 这个 mode 是没有 lite 配置项的，lite 模式即将废弃，参考 [lite 模式废弃说明](./options-v2.mdx#lite-模式废弃说明)。
+
+## options
+
+<Badge text="V1.2.4 新增 API" type="tip" />
+
+- **类型：** `BriefModeOptions | NormalModeOptions`
+- **可选：** `true`
+- **默认值：** `undefined`
+- **描述：**
+  根据不同的 `mode` 配置，`options` 支持不同的配置项：
+
+### mode: 'brief'
+
+Brief 模式用于生成轻量级的分析报告，支持 HTML 和 JSON 两种输出格式。
+
+- **`type`**：输出类型，支持 `'html'` 和 `'json'` 数组，可同时选择多种格式
+- **`htmlOptions`**：HTML 输出相关配置
+  - **`reportHtmlName`**：HTML 报告文件名，默认为 `report-rsdoctor.html`
+- **`jsonOptions`**：JSON 输出相关配置
+  - **`sections`**：JSON 输出包含的模块配置
+    - **`moduleGraph`**：是否包含模块图数据，默认为 `true`
+    - **`chunkGraph`**：是否包含 Chunk 图数据，默认为 `true`
+    - **`rules`**：是否包含规则数据，默认为 `true`
+
+**配置示例：**
+
+```js
+new RsdoctorRspackPlugin({
+  output: {
+    mode: 'brief',
+    options: {
+      // 同时输出 HTML 和 JSON 格式
+      type: ['html', 'json'],
+      htmlOptions: {
+        reportHtmlName: 'my-report.html',
+      },
+    },
+  },
+});
+```
+
+**注意事项：**
+
+- Brief 模式下，**`reportCodeType`** 配置项无效，因为只支持 **`'noCode'`**。
+- Brief 模式下，**type 为 'html' 时**，分析数据会直接注入到 HTML 文件中，便于单文件分享。
+- Brief 模式下，**type 为 'json' 时**，会生成结构化的数据文件，便于程序化处理。
+
+### mode: 'normal'
+
+Normal 模式下的 `options` 为空对象，无额外配置项。
+
+### 类型定义
+
+#### BriefModeOptions
+
+```ts
+interface BriefModeOptions {
+  /** 输出类型，支持 HTML 和 JSON */
+  type?: Array<'html' | 'json'>;
+  /** HTML 输出相关配置 */
+  htmlOptions?: BriefConfig;
+  /** JSON 输出相关配置 */
+  jsonOptions?: JsonOptions;
+}
+
+interface BriefConfig {
+  /** HTML 报告文件名，默认为 report-rsdoctor.html */
+  reportHtmlName?: string;
+  /** 是否额外生成 JSON 数据文件，默认为 false */
+  writeDataJson: boolean;
+}
+
+interface JsonOptions {
+  /** JSON 输出包含的模块配置 */
+  sections?: JsonSectionOptions;
+}
+
+interface JsonSectionOptions {
+  /** 是否包含模块图数据，默认为 true */
+  moduleGraph?: boolean;
+  /** 是否包含 Chunk 图数据，默认为 true */
+  chunkGraph?: boolean;
+  /** 是否包含规则数据，默认为 true */
+  rules?: boolean;
+}
+```
+
+#### NormalModeOptions
+
+NormalModeOptions 目前类型为空 Object。
+
+## reportCodeType
+
+- **类型：** `"noModuleSource" | "noAssetsAndModuleSource" | "noCode" | { noModuleSource?: boolean; noAssetsAndModuleSource?: boolean; noCode?: boolean }`
+  - **V2 时，类型将只支持 String 类型。**
+- **可选：** `true`
+- **默认值：** `undefined`，undefined 时输出所有完整数据。
+- **描述：**
+  选择输出的分析数据范围：
+  - **默认**：输出所有完整数据
+  - **`noModuleSource`**：输出除模块（[Module](https://rspack.rs/config/module#module)）代码外的数据，模块（[Module](https://rspack.rs/config/module#module)）代码即产物（（[Module](https://rspack.rs/config/module#module)））内各个文件的打包模块代码
+  - **`noAssetsAndModuleSource`**：输出除模块（[Module](https://rspack.rs/config/module#module)）代码和 Assets 产物代码外的数据
+  - **`noCode`**：输出除代码外的数据，即不输出任何代码
+
+  数据输出从丰富到简的排序：`noModuleSource` > `noAssetsAndModuleSource` > `noCode`。
```

Comment:
it's better to create a table to show when to use each mode. it's difficult for users to choose one, even when they are informed about their contents.